### ISSUE TITLE
Tracker issue for 4.13 readiness

### ISSUE THREAD

**Issue by praveenkumar** [crc-org/snc#652]
This issue is to track any Jira/BZ which are filed against OCP-4.13 and can block crc bundle creation. We will also incorporate changes which are only for release-4.13 ( like adding more capabilities)

- [ ] https://issues.redhat.com/browse/OCPBUGS-7864

**Comment by adrianriobo** [comment]
We are seeing a consistent error when building the bundle for `4.13.0-ec.3` this is reproduced on every execution:

```bash
+ '[' 9 -lt 10 ']'
+ echo 'Retry 9/10 exited 1, retrying in 256 seconds...'
Retry 9/10 exited 1, retrying in 256 seconds...
+ sleep 256
+ all_pods_are_running_completed none
+ local ignoreNamespace=none
+ ./openshift-clients/linux/oc get pod --no-headers --all-namespaces '--field-selector=metadata.namespace!=none'
+ grep -v Running
+ grep -v Completed
openshift-kube-apiserver                           kube-apiserver-startup-monitor-crc-gj9lv-master-0         0/1   CrashLoopBackOff   7 (119s ago)   12m
+ exit=1
+ wait=512
+ count=10
+ '[' 10 -lt 10 ']'
+ echo 'Retry 10/10 exited 1, no more retries left.'
Retry 10/10 exited 1, no more retries left.
```

Everytime exact same pod errors with `CrashLoopBackOff`

Will try to debug to get the logs from the pod

**Comment by praveenkumar** [comment]
There is `4.13.0-ec.4` bits now available, better to check with it if bundle creation still fails?

**Comment by praveenkumar** [comment]
closing it since we released with 4.13 bundle.
